Gemfile

```diff
@@ -53,6 +53,11 @@ gem 'sshkey', '~> 1.9'
 gem 'dynflow', '>= 0.8.29', '< 1.0.0'
 gem 'daemons'
 gem 'get_process_mem'
+gem 'graphql', '~> 1.7.8'
+gem 'graphql-activerecord'
+gem 'graphql-batch'
+gem 'jwt', '~> 2.1.0'
```

Comment:
The commit message should mention `http://projects.theforeman.org/issues/22388` if we add this in a single PR.

`Fixes #22109,#22388 - add graphql as api`